tests/unittests/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) Microsoft. All rights reserved
+#
+
+include $(CLEAR_VARS)
+
+CLANG := clang++
+
+SRCDIR := Foundation
+SRCFILES := $(wildcard $(SRCDIR)/*.m*)
+RF_SRCFILES := $(wildcard $(SRCDIR)/ReferenceFoundation/*.mm)
```

Comment:
I'm doing something separate with them (ReferenceFoundation output files have a ReferenceFoundation_ prefix to avoid name clashes)

---
In reply to: [70349591](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70349591) [](ancestors = 70349591)